src/main/java/org/kiwiproject/dropwizard/util/config/JobSchedule.java

```diff
@@ -0,0 +1,26 @@
+package org.kiwiproject.dropwizard.util.config;
+
+import io.dropwizard.util.Duration;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Config object that defines a schedule for background jobs.
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class JobSchedule {
+
+    @NotNull
+    private Duration initialDelay = Duration.seconds(30);
+
+    @NotNull
+    private Duration intervalDelay;
```

Comment:
We need to document the fact that this only supports a fixed delay from the last occurrence. (I am correct in thinking that is what we do right?)

Reply:
The interval technically can be fixed rate or delay from this